common/beerocks/bcl/source/network/socket.cpp

```diff
@@ -594,3 +596,168 @@ size_t Socket::getBytesWritePending()
     return (size_t)pending;
 }
 #endif
+
+SocketPoll::SocketPoll(uint timeout) : m_epoll_fd(-1), m_timeout(timeout)
+{
+    m_sockets_map.clear();
+    if ((m_epoll_fd = epoll_create1(0)) == -1) {
+        LOG(ERROR) << "Failed creating epoll: " << strerror(errno);
+    }
```

Comment:
I can't imagine any situation where we could have this error and usefully continue, so I'd change this into an assert:
```suggestion
    m_epoll_fd = epoll_create1(0);
    LOG_IF(FATAL, m_epoll_fd == -1) << "Failed creating epoll: " << strerror(errno);
```

Reply:
Done